Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "mediatek-brom"
-version = "0.1.0"
+version = "1.0.0"
```

```suggestion
version = "0.2.0"
```
Not ready to release version 1 just yet :)  Bumping the first non-zero component  (the minor in this case) indicates an incompatible version

---

Ah, i see, from https://semver.org/:
```
4. Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.
5. Version 1.0.0 defines the public API. The way in which the version number is incremented after this release is dependent on this public API and how it changes.
```